CONTRIBUTING.md

```diff
@@ -195,6 +197,8 @@ Note the newline separating the body from the footer, as well as between the JIR
 ```
 <BODY>
 
+fixes: #123
```

Comment:
Wanna add this to the example that follows for good measure?

Reply:
👍 Done in ec7fa17936c46a247f97c31784ff1c91ab6701d0